### PR TITLE
[MAT-6576] Use Spotify Formatter

### PR DESCRIPTION
**Pull request by jkotanchik-SB**
## Terminology Service PR

Jira Ticket: [MAT-6576](https://jira.cms.gov/browse/MAT-6576)
(Optional) Related Tickets:

### Summary

Use Spotify formatter since it supports newer Java feature syntax.

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included.
* [ ] No extemporaneous files are included (i.e Complied files or testing results).
* [ ] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

**Comment by jkotanchik-SB** [comment]
Gitleaks passes but then spends 30m trying to "clean up orphaned processes".  See [Gitleaks run](https://github.com/MeasureAuthoringTool/terminology-service/actions/runs/9176750139/job/25234007193?pr=80).

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -40,8 +40,8 @@ spring:
     max-in-memory-size: 24MB
   data:
     mongodb:
-      uri: ${MONGO_URI:mongodb://${DBUSER:root}:${DBPASS:E5press0}@localhost:27017/terminology}?authSource=admin&maxPoolSize=50&connectTimeoutMS=2000&serverSelectionTimeoutMS=2000
+      uri: mongodb://root:E5press0@${MONGO_HOST:localhost}:27017/madie?authSource=admin&maxPoolSize=50&connectTimeoutMS=2000&serverSelectionTimeoutMS=2000
```
Comment (nmorasb) [inline src/main/resources/application.yml]: was this intentionally added? Should we maybe make `root` a default value for the `${DBUSER}` var? (And same for `DBPASS`)

Comment (jkotanchik-SB) [reply src/main/resources/application.yml]: Nope! Accidental add. I'll clear it out.

**Review by chubert-sb — APPROVED** [review]
